[BUG] CircleCi tests failing because Maven now requires https downloads

#### Bug Report Checklist

- [ ] Have you provided a full/minimal spec to reproduce the issue?
- [ ] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [ ] What's the version of OpenAPI Generator used?
- [ ] Have you search for related issues/PRs?
- [ ] What's the actual output vs expected output?
- [ ] [Optional] Bounty to sponsor the fix ([example](https://www.bountysource.com/issues/66123212-javascript-client-produces-a-wrong-object-for-a-string-enum-type-that-is-used-with-ref))

##### Description

CircleCi tests failing because Maven now requires https downloads
See a failure here: https://circleci.com/gh/OpenAPITools/openapi-generator/11802#tests/containers/1
Our error message is:
```
Download https://repo.jfrog.org/artifactory/gradle-plugins/org/codehaus/jackson/jackson-core-asl/1.5.1/jackson-core-asl-1.5.1.jar
:compileJava UP-TO-DATE
:compileGroovy

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':compile'.
> Could not resolve org.codehaus.groovy:groovy-all:2.5.7.
  Required by:
      org.openapitools:groovy:1.0.0
   > Could not GET 'http://repo1.maven.org/maven2/org/codehaus/groovy/groovy-all/2.5.7/groovy-all-2.5.7.pom'. Received status code 501 from server: HTTPS Required
> Could not resolve io.github.http-builder-ng:http-builder-ng-core:1.0.3.
  Required by:
      org.openapitools:groovy:1.0.0
   > Could not GET 'http://repo1.maven.org/maven2/io/github/http-builder-ng/http-builder-ng-core/1.0.3/http-builder-ng-core-1.0.3.pom'. Received status code 501 from server: HTTPS Required

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 13.178 secs
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.2.1:exec (bundle-test) on project GroovyPetstoreClientTests: Command execution failed. Process exited with an error: 1 (Exit value: 1) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :GroovyPetstoreClientTests
```

##### openapi-generator version

master branch

##### Suggest a fix

Update the url that maven is using to use a https prefix

--- a/modules/openapi-generator/src/main/resources/Groovy/build.gradle.mustache
+++ b/modules/openapi-generator/src/main/resources/Groovy/build.gradle.mustache
@@ -13,6 +13,7 @@ wrapper {
 
 buildscript {
     repositories {
+        maven { url "https://repo1.maven.org/maven2" }
         maven { url 'https://repo.jfrog.org/artifactory/gradle-plugins' }
     }
     dependencies {

--- a/modules/openapi-generator/src/main/resources/java-pkmst/pom.mustache
+++ b/modules/openapi-generator/src/main/resources/java-pkmst/pom.mustache
@@ -47,7 +47,7 @@
          <id>central</id>
          <name>Maven Repository Switchboard</name>
          <layout>default</layout>
-         <url>http://repo1.maven.org/maven2</url>
+         <url>https://repo1.maven.org/maven2</url>
          <snapshots>
             <enabled>false</enabled>
          </snapshots>
@@ -299,7 +299,7 @@
                      </excludes>
                   </configuration>
                </plugin>
-               
+
                <plugin>
                   <groupId>net.alchim31.maven</groupId>
                   <artifactId>scala-maven-plugin</artifactId>
@@ -315,7 +315,7 @@
                             <runMultipleSimulations>true</runMultipleSimulations>
                             <includes>
                                 <simulationClass>testapi</simulationClass>
-                                
+
                             </includes>
                         </configuration>
                         <executions>

--- a/samples/client/petstore/groovy/build.gradle
+++ b/samples/client/petstore/groovy/build.gradle
@@ -13,6 +13,7 @@ wrapper {
 
 buildscript {
     repositories {
+        maven { url "https://repo1.maven.org/maven2" }
         maven { url 'https://repo.jfrog.org/artifactory/gradle-plugins' }
     }
     dependencies {

--- a/samples/server/petstore/java-pkmst/pom.xml
+++ b/samples/server/petstore/java-pkmst/pom.xml
@@ -47,7 +47,7 @@
          <id>central</id>
          <name>Maven Repository Switchboard</name>
          <layout>default</layout>
-         <url>http://repo1.maven.org/maven2</url>
+         <url>https://repo1.maven.org/maven2</url>
          <snapshots>
             <enabled>false</enabled>
          </snapshots>
@@ -268,7 +268,7 @@
                      </excludes>
                   </configuration>
                </plugin>
-               
+
                <plugin>
                   <groupId>net.alchim31.maven</groupId>
                   <artifactId>scala-maven-plugin</artifactId>
@@ -284,7 +284,7 @@
                             <runMultipleSimulations>true</runMultipleSimulations>
                             <includes>
                                 <simulationClass>testapi</simulationClass>
-                                
+
                             </includes>
                         </configuration>
                         <executions>

